ui/app/src/components/HistoryDialog/Pagination.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2007-2021 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useIntl } from 'react-intl';
+import TablePagination from '@material-ui/core/TablePagination';
+import translations from './translations';
+import React from 'react';
+import { paginationStyles } from './HistoryDialog';
+import { PaginationProps } from './utils';
+
+export function Pagination(props: PaginationProps) {
```

Comment:
Is this any different from the generic "Pagination" component we have somewhere? Could we sub?

Reply:
I added a TODO